types/es-abstract/.editorconfig

```diff
@@ -0,0 +1,3 @@
+# This package uses tabs
```

Comment:
Don't override the root .editorconfig, thanks. We really do want all definitions to conform to that global style guideline. It makes both review and contribution much easier. ❤️

Reply:
@sandersn, @PranavSenthilnathan, @andrewbranch and&nbsp;@armanio123 have&nbsp;all&nbsp;allowed me&nbsp;to&nbsp;keep&nbsp;using the&nbsp;superior&nbsp;tabs&nbsp;in:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34364#discussion_r270972375, https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34894#discussion_r279131192, https://github.com/DefinitelyTyped/DefinitelyTyped/pull/39055#issuecomment-542370662, https://github.com/DefinitelyTyped/DefinitelyTyped/pull/38947#pullrequestreview-303558276

Reply:
Yep, and that ends today. We've had a talk, and no - just because someone (mistakenly) allowed a commit with what amounts to a style rule suppression, doesn't mean we're going to keep allowing it.

Reply:
The root `.editorconfig` exists quite literally so I shouldn't have to have this discussion BTW - your insistence is going to force us to update our tooling to forbid nested `.editorconfig`s. Congratulations on being a catalyst for change~